@here/harp-mapview/lib/MapView.ts

```diff
@@ -1812,6 +1817,20 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_mapAnchors;
     }
 
+    /**
+     * The node in this MapView's scene containing the overlayed user [[MapAnchor]]s.
+     *
+     * @remarks
+     * The `MapAnchor`s added to `mapAnchorsOverlay` are rendered on top of the MapView scene.
+     *
+     * All (first level) children of this node will be positioned in world space according to the
+     * [[MapAnchor.geoPosition]].
+     * Deeper level children can be used to position custom objects relative to the anchor node.
+     */
```

Comment:
would it make sense to add a @beta? here

Reply:
done.

Reply:
Now we have 3 anchor APIs: `mapAnchors`, `worldAnchors` and `mapAnchorsOverlay`.
Should be merged into one api (e.g. `mapObjects`) with following properties:
- anchor (world or geo space)
- overlay

Nevertheless it's fine for now to have it as a temporary API but we should clean it up asap.

Reply:
i would use just `MapAnchor` or `Anchor`. `MapObject` may not be the correct name, in fact `MapObject` was the initial name we used for this API but then we replaced it with MapAnchor.
https://github.com/heremaps/harp.gl/pull/530